packages/react/src/components/iconBadge/IconBadge.tsx

```diff
@@ -31,7 +29,7 @@ export interface IconBadgeProps {
      */
     size?: IconBadgeSize;
     /**
-     * Additionnal CSS class for the icon
+     * @deprecated will have no effet
      */
     svgClass?: string;
```

Comment:
Just remove it, you're already making a breaking change by changing `svgName` to `icon`